README.md

```diff
@@ -100,6 +102,8 @@ where
 (this is basically the output of the MLC method)
 - ``true_labels`` is a comma-separated list of true labels (ground truth)
 
+Note that you can omit the ``true_labels`` section if you only want to predict the quality of the data.
```

Comment:
"quality of the output"? otherwise will be confused with training data?

Reply:
Perhaps be more precise: "quality of the MLC output"?

Reply:
okay